src/video_emotion/extract_classifier.py

```diff
@@ -45,7 +62,37 @@ def classify_video(video_path, time_range=None):
         "surprise": surprise_sum / number_of_faces,
         "neutral": neutral_sum / number_of_faces
     }
+def find_emotions_cutoffs(emotions):
+
+    """Finds the emotional cutoffs for a set of emotions as a list
+
+        Parameters
+        ----------
+        emotions
+            The path of the video that is to be analyzedemotions
```

Comment:
Typo here. Should probably be changed to "The path of the video that is to be analyzed"

Also, if it's a path, can't you give it the type string?